exchanges/ftx/ftx_wrapper.go

```diff
@@ -0,0 +1,874 @@
+			tempResp.Amount = triggerOrderData.Result[z].Size
+			tempResp.AssetType = assetType
+			tempResp.Date = triggerOrderData.Result[z].CreatedAt
+			tempResp.Exchange = f.Name
+			tempResp.ExecutedAmount = triggerOrderData.Result[z].FilledSize
+			tempResp.Pair = currency.NewPairFromString(triggerOrderData.Result[z].Market)
+			tempResp.Price = triggerOrderData.Result[z].AvgFillPrice
+			tempResp.RemainingAmount = triggerOrderData.Result[z].Size - triggerOrderData.Result[z].FilledSize
+			tempResp.TriggerPrice = triggerOrderData.Result[z].TriggerPrice
+			orderVars, err := f.compatibleOrderVars(triggerOrderData.Result[z].Side,
+				triggerOrderData.Result[z].Status,
+				triggerOrderData.Result[z].OrderType,
+				triggerOrderData.Result[z].FilledSize,
+				triggerOrderData.Result[z].Size,
+				triggerOrderData.Result[z].AvgFillPrice)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.Status = orderVars.Status
+			tempResp.Side = orderVars.Side
+			tempResp.Type = orderVars.OrderType
+			tempResp.Fee = orderVars.Fee
+			resp = append(resp, tempResp)
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderHistory retrieves account order information
+// Can Limit response to specific order status
+func (f *FTX) GetOrderHistory(getOrdersRequest *order.GetOrdersRequest) ([]order.Detail, error) {
+	var resp []order.Detail
+	for x := range getOrdersRequest.Pairs {
+		var tempResp order.Detail
+		assetType, err := f.GetPairAssetType(getOrdersRequest.Pairs[x])
+		if err != nil {
+			return resp, err
+		}
+		orderData, err := f.FetchOrderHistory(f.FormatExchangeCurrency(getOrdersRequest.Pairs[x], assetType).String(),
+			getOrdersRequest.StartTicks, getOrdersRequest.EndTicks, "")
+		if err != nil {
+			return resp, err
+		}
+		for y := range orderData.Result {
+			tempResp.ID = strconv.FormatInt(orderData.Result[y].ID, 10)
+			tempResp.Amount = orderData.Result[y].Size
+			tempResp.AssetType = assetType
+			tempResp.ClientOrderID = orderData.Result[y].ClientID
+			tempResp.Date = orderData.Result[y].CreatedAt
+			tempResp.Exchange = f.Name
+			tempResp.ExecutedAmount = orderData.Result[y].Size - orderData.Result[y].RemainingSize
+			tempResp.Pair = currency.NewPairFromString(orderData.Result[y].Market)
+			tempResp.Price = orderData.Result[y].Price
+			tempResp.RemainingAmount = orderData.Result[y].RemainingSize
+			var orderVars OrderVars
+			orderVars, err = f.compatibleOrderVars(orderData.Result[y].Side,
+				orderData.Result[y].Status,
+				orderData.Result[y].OrderType,
+				orderData.Result[y].FilledSize,
+				orderData.Result[y].Size,
+				orderData.Result[y].AvgFillPrice)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.Status = orderVars.Status
+			tempResp.Side = orderVars.Side
+			tempResp.Type = orderVars.OrderType
+			tempResp.Fee = orderVars.Fee
+			resp = append(resp, tempResp)
+		}
+		triggerOrderData, err := f.GetTriggerOrderHistory(f.FormatExchangeCurrency(getOrdersRequest.Pairs[x], assetType).String(),
```

Comment:
This is set incorrectly. The type  strings.ToLower(getOrdersRequest.Type.String()) aka `limit` or `market` is not the type that FTX is searching for. The type you want to set with limit/market is the `orderType` whereas you are setting is as the `type` per https://docs.ftx.com/#get-trigger-order-triggers
I think I'm botching my explanation, so I'll write it twice. You are assigning the ordertype to the wrong FTX parameter. If you look at `TestGetTriggerOrderHistory` you set the type to `stop` which is the parameter `type` where the ordertypes should be assigned to the parameter `orderType`